src/cargo-fmt/main.rs

```diff
@@ -57,6 +58,14 @@ pub struct Opts {
     /// Format all packages (only usable in workspaces)
     #[structopt(long = "all")]
     format_all: bool,
+
+    /// Format files located within subdirectories of `tests` directory
+    #[structopt(long = "include-nested-test-files")]
+    include_nested_test_files: bool,
+
+    /// Files within subdirectories of `tests` directory to ignore
+    #[structopt(long = "ignored-nested-test-files")]
+    ignored_nested_test_files: Vec<String>,
```

Comment:
Not convinced these are the best names for these opts so please let me know if anyone has any suggestions (they could certainly use a short version too!)